src/config.rs

```diff
@@ -746,20 +753,20 @@ impl Config {
 
         let section = match &self.section {
             Some(section) => section.as_str(),
-            None => bail!("key '{}' does not belong to a section", key),
+            None => bail!(tr!("key '{}' does not belong to a section", key)),
         };
 
         match section {
             "options" => self.parse_option(key, value),
             "bin" => self.parse_bin(key, value),
-            _ => bail!("unknown section '{}', section"),
+            _ => bail!(tr!("unknown section '{}', section")),
```

Comment:
```suggestion
            _ => bail!(tr!("unknown section '{}'"), section),
```

Reply:
Looks like the tr macro doesn't panic if there's the amount of `{}` and values don't match up. That's unfortunate.